javascript-loops-practice/get-indexes.js

```diff
@@ -0,0 +1,8 @@
+/* exported getIndexes */
+function getIndexes(array) {
+  var newArr = [];
+  for (var i = 0; i < array.length; i++) {
+    newArr.push(array.indexOf(array[i]));
```

Comment:
You can just use `i`

Reply:
![image](https://user-images.githubusercontent.com/117477893/208754411-064e0035-f3a0-44af-ad2c-a486ff69fb8f.png)